src/crawlee/_utils/html_to_text.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
```

Comment:
I would prefer moving this to e.g. `crawlee/beautifulsoup_crawler/_utils.py` to avoid the need for try-except imports here.

Reply:
on top of that, we have the PW's `infinite_scroll` helper there following the same structure